microshift_configuring/microshift-using-config-yaml.adoc

```diff
@@ -29,3 +29,7 @@ include::modules/microshift-config-nodeport-limits.adoc[leveloffset=+2]
 == Additional resources
 
 * xref:../../microshift-greenboot-checking-status.adoc#microshift-greenboot-checking-status[Checking Greenboot status]
+
+* xref:../../microshift-ingress-controller-conc.adoc#microshift-ingress-control-concept_{context}[Using ingress control in {microshift-short}]
```

Comment:
```suggestion
* xref:../microshift-ingress-controller.adoc#microshift-ingress-controller-concept_microshift-ingress-controller[Using ingress control in {microshift-short}]
```
try this if you want to reference the concept module; you need `assembly.adoc#moduleID_assemblyID[title]`
and this xref is one level deep (`../`) because the ingress assembly is in the same directory as the config assembly (you count the levels from where you start, which is where the xref is, does that make sense?)